package.json

```
@@ -40,7 +40,7 @@
     "test": "./test"
   },
   "engines": {
-    "node": ">= 14.0.0"
+    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
```

TypeScript still supports Node 14.17. I asked about their plans in https://discord.com/channels/508357248330760243/640177505915109377/1293239882323591200.

---

Confirmed we're 👍 to break TypeScript.

Related: @voxpelli noted in a private chat that bumping from 14 to 20 is _two_ even major version bumps. And (still quoting here) it could possibly also be that since Node 18 introduced nodes test runner, that a maintained test runner for older versions are in extra high demand.

👍 agreement from me - it'd be nice to ease into updating version requirements.

```suggestion
    "node": "^16.16.0 || ^18.18.0 || ^20.9.0 || >=21.1.0"
```